go.mod

```diff
@@ -351,3 +349,9 @@ require (
 )
 
 require github.com/jackpal/gateway v1.0.10
+
+require (
+	github.com/ajeddeloh/yaml v0.0.0-20170912190910-6b94386aeefd // indirect
+	github.com/coreos/container-linux-config-transpiler v0.9.0 // indirect
+	github.com/coreos/ignition v0.35.0 // indirect
+)
```

Comment:
Please merge this into the existing require() blocks for indirect dependencies.

Reply:
Done